libcontainer/mount/mount.go

```diff
@@ -1,55 +0,0 @@
-package mount
```

Comment:
Not too much, given that we are not syncing fully with runc anymore. Rather we are just cherry-picking changes in runc that apply to Sysbox. The only thing that is changing is the location of the mount package, so it's not too hard to handle. I did this because sysbox-mgr needs to access this package too, and if I leave it in sysbox-runc I end up with circular dependencies in the Golang imports.

Reply:
> Not too much, given that we are not syncing fully with runc anymore

I think that sooner or later we will need to sync up to grab a critical feature that relies on multiple changes, and then this task will be much more difficult to complete. If/when that happens we can reassess if it's worth moving the files back and dealing with the dependency-cycles problem in a different way.